### DISCUSSION TITLE
Abandoned & Missing Content

### DISCUSSION THREAD

**Discussion by johko** [huggingface/computer-vision-course#185] [Ideas]
Please post here if there is content from your group that is still missing or people stopped working on it. This also applies if people of your group stopped responding. 

This way we can get a better overview of topics that are still to be processed.

**Comment by charchit7** [comment]
Hey @johko this is the current work in progress PR, it's almost done. It's just I had a cut in my right hand so not working properly : https://github.com/johko/computer-vision-course/pull/175

**Comment by seshupavan** [comment]
Hey everyone, we are working unit-0: welcome to the course so checking the repository and found the commits for units- 6,7 & 11 are quite low compared to others and there are less files. Can we know the reason? Is it because of inactive team? Busy schedules? - knowing what's going on helps to find a better way to move this forward.

Unit - 6: Basic CV tasks --> @Skower , @kaustubh-s1, @sarthak247, @adhiiisetiawan 
Unit 7 - Video and Video Processing --> @DiwakarBasnet @sazio @sanae-a11y @jungnerd
 Unit 11 - Zero-shot Computer Vision --> @mmhamdy, @albertkao227, @Maddy011

**Reply by adhiiisetiawan** [reply]
hi, i'm from unit 6. i with @Skower working on segmentation part. and yeah, most of our team like busy and i can't reach them on discord. i still try to reach them until tommorow. maybe if until 1 or 2 days not answered, we can free up the space.

also we have new member, working on object detection @abdullah-alnahas. i will try to reach them first

**Reply by DiwakarBasnet** [reply]
hello, i'm from unit 7. I was working on vivit model but haven't been able to lately because of exams   and i can't reach my team members on discord.

**Reply by wo0jun** [reply]
hi, I'm from unit 7. I've been working on a key application, but I've been busy getting into grad school. 
Me and @DiwakarBasnet will try to reach out to our teammates again.

**Reply by mmhamdy** [reply]
Hi @seshu-pavan. Unit 11 here. The only active members in this group are me and @albertkao227. The 2nd section is almost done, and the 3rd and last 2 are WIP. other than that, are unassigned which I'm planning to work on once I finish WIP.

**Reply by miniMaddy** [reply]
Hi, @adhiiisetiawan , I would love to work on the Basic CV tasks, can you please add me to your group if you still need help. 
Discord username: @skittle

**Reply by adhiiisetiawan** [reply]
hi @miniMaddy , sure.

**Comment by sitammeur** [comment]
@johko For transfer learning, contributors are added to the sheet, but then everybody is inactive. So, I then forked the repo and made a PR. Commit a few changes, and it will be merged. I have found some contributors who are willing to help, so we will try to contribute actively. Also, as I am connected with another chapter, even though I am busy with that, this chapter contribution will still be continuous. Once finalized, I will edit this message and tag all the contributors for the transfer learning chapter.

**Reply by seshupavan** [reply]
@bellabf and @aman06012003 showed their interest in working on transfer learning so pinging them here. 🤗
